linker/linker.go

```diff
@@ -114,13 +121,22 @@ func (l Linker) DoLink(args []string) (count int) {
 	if len(libs) > 0 || len(files) > 0 {
 
 		audit := audit.New(l.ossiConfig)
-		return audit.ProcessPaths(
-			iterateAndAppendToLibPathsSlice(libPaths),
-			iterateAndAppendToSlice(libs),
-			iterateAndAppendToSlice(files))
+		libPaths := iterateAndAppendToLibPathsSlice(libPaths)
+		libs := iterateAndAppendToSlice(libs)
+		files := iterateAndAppendToSlice(files)
+		count := audit.ProcessPaths(
+			libPaths,
+			libs,
+			files)
+		return &Results {
+			LibPaths: libPaths,
+			Libs: libs,
+			Files: files,
+			Count: count,
+		}
 	}
 
-	return 0
+	return new(Results)
```

Comment:
Can you do `return &Results{}`? Does the same thing I think.

Reply:
yeah i just found this a bit easier to understand. i saw it when i was deep diving go langs list implementation. :)

Reply:
Cool deal! I'm used to the other way, but don't let this old crufty nut sway you :)